### DISCUSSION TITLE
Sky View Factor factor arguments "distance_units", "area_units", and "ellipsoid"

### DISCUSSION THREAD

**Discussion by jlegewie** [UMEP-dev/UMEP#567] [General]
Hi,

we are unable to find any documentation for the `distance_units`, `area_units` and `ellipsoid` arguments of the "umep:Urban Geometry: Sky View Factor" process. The GUI doesn't show them but they appear when using "Advanced -> Copy as qgis_process command" as `--distance_units=meters --area_units=m2 --ellipsoid=EPSG:7019`.

- Is there any documentation of these arguments that we missed?
- Where does EPSG 7019 come from? Our rasters are in 2263. 
- The unit of our CRS (EPSG 2263) and height measure for our DSM is feet. Can we change the arguments to something like `--distance_units=feet --area_units=ft2 --ellipsoid=EPSG:2263`?

Thanks

**Comment by biglimp** [comment]
Hi,
Those parameters should be needed. There seem to be QGIS that add them. Just copy the syntax from the History button in the Processing Toolbox. Below is an example:
`processing.run("umep:Urban Geometry: Sky View Factor", {'INPUT_DSM':'C:/Users/xlinfr/Documents/PythonScripts/SOLWEIG/SOLWEIGdata/DSM_KRbig.tif','INPUT_CDSM':None,'TRANS_VEG':3,'INPUT_TDSM':None,'INPUT_THEIGHT':25,'ANISO':True,'OUTPUT_DIR':'TEMPORARY_OUTPUT','OUTPUT_FILE':'TEMPORARY_OUTPUT'})`

**Comment by jlegewie** [comment]
Thanks! I assume that is also true for "Wall Height and Aspect"? In that case, the documentation states that the DSM should use a CRS based on meters as units. Is it possible to use "Wall Height and Aspect" with a DSM raster that uses feet at the unit of height and as the unit for the CRS?

**Reply by biglimp** [reply]
We did not fully test a with a CRS that is using e.g. feet. Please test and see what kind of result you get compared to a meter-based CRS. I suspect that "Wall Height and Aspect" will produce similar results but SVF Calculator would should differences within the two CRSs.
